galaxy.yml

```diff
@@ -71,7 +71,7 @@ tags:
 # L(specifiers,https://python-semanticversion.readthedocs.io/en/latest/#requirement-specification). Multiple version
 # range specifiers can be set and are separated by ','
 dependencies:
-  'git+https://github.com/containers/ansible-podman-collections': 'master'
+  'git+https://github.com/containers/ansible-podman-collections': 'main'
```

Comment:
I think if you need to re-sync the fork and rebase on main. This change already merged.